### ISSUE TITLE
`Math.f32round` alias?

### ISSUE THREAD

**Issue by zloirock** [tc39/proposal-float16array#10]
It's strange to have `Math.f16round` for Float16 and `Math.fround` for Float32 in terms of naming.

Maybe makes sense to add `Math.f32round` alias for consistency?

**Comment by bakkot** [comment]
I don't think it's that weird. In C++ (for example) "float" means binary32 floats, and "__fp16" means binary16.

Very few people need this, and those that do are likely to understand the distinction, so I don't think it's worth having an alias. I'll leave this open so others can chime in, but I doubt we'll do this.

**Comment by zloirock** [comment]
It's JS, not C++, and the base number type is number -> float64, so there by the name "float" I see float64. Clarification is required, rather, for float32 that's used much less often.

**Comment by annevk** [comment]
I agree with @bakkot. This can always be added once there's a clear need established for it through Stack Overflow questions or libraries offering a wrapper.

**Comment by bakkot** [comment]
Proposal landed without this. If there's need it can be done in a separate proposal, but I don't foresee this coming up.
